google-cloud-tools-plugin/google-cloud-storage/src/com/google/cloud/tools/intellij/gcs/GoogleCloudStorageMessageBundle.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.gcs;
+
+import com.intellij.CommonBundle;
+import java.util.ResourceBundle;
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.PropertyKey;
+
+public class GoogleCloudStorageMessageBundle {
```

Comment:
nit: but for consistency with `GctMessageBundle`, maybe `GcsMessageBundle`? I'm fine renaming the other one too.

Reply:
I'm being consistent with core. GCT should cease to exist when all the modules are extracted out.

Reply:
sgtm